mmv1/products/dataplex/Glossary.yaml

```diff
@@ -0,0 +1,113 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Glossary'
+description: |
+  Represents a collection of categories and terms defined by the user. Glossary is a top level resource and is the GCP parent resource of all the categories and terms within it.
+base_url: 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
```

Comment:
```suggestion
min_version: beta
base_url: 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
```

Reply:
per discussion - we had a miscommunication earlier, and this is not a Public Preview resource.